### PR TITLE
#14111 [Robosaur] Rex project creation configuration

### PR DESCRIPTION
**Pull request by michael-jong-ds**
1. Added config for project creation with custom OCR API and ML assisted labeling

### CODE REVIEW

**Comment by tonoman3g** [comment]
please create a new branch from `master`  named `rex`

then create PR to `rex`

we are going to develop this in `rex` branch

@michael-jong-ds

**Comment by michael-jong-ds** [comment]
> Please create a new branch from `master` named `rex`.
> 
> Then create PR to `rex`.
> 
> We are going to develop this in `rex` branch.
> 
> @michael-jong-ds

changed the base branch kak

File: sample/SI/config_si.json [hunk 1]
```diff
@@ -0,0 +1,214 @@
+{
```
Comment (tonoman3g) [inline sample/SI/config_si.json:1]: please don't use SI as the folder name here

`sample/rex/config_rex.json`

File: sample/SI/config_si.json [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+          ]
+        }
+      ],
+      "documentAssignments": [
+        {
+          "teamMemberId": "<TEAM_MEMBER_ID>",
+          "role": "LABELER_AND_REVIEWER"
+        }
+      ],
+      "tagNames": ["<TAG_1>", "<TAG_2>"],
```
Comment (tonoman3g) [inline sample/SI/config_si.json:202]: should we put the `SI` tag here? @jonathancesario

File: sample/SI/config_si.json [hunk 3]
```diff
@@ -0,0 +1,214 @@
...
+      "kinds": ["TOKEN_BASED"],
+      "type": "OCR"
+    },
+    "autoLabel": {
+      "enableAutoLabel": true,
+      "labelerEmail": "<EMAIL>",
```
Comment (tonoman3g) [inline sample/SI/config_si.json:208]: what is the use of `labelerEmail` field here? @michael-jong-ds

Comment (michael-jong-ds) [reply sample/SI/config_si.json:208]: it's needed by the mutation
```
input AutoLabelTokenBasedProjectInput {
  projectId: ID!
  labelerEmail: String!
  targetAPI: TargetApiInput!
  options: AutoLabelProjectOptionsInput!
}
```

**Review by tonoman3g — APPROVED** [review]

File: sample/SI/config_si.json (hunk 2 above)
Comment (jonathancesario) [reply sample/SI/config_si.json]: like what? is there any sample that we can use?

Comment (tonoman3g) [reply sample/SI/config_si.json]: like this, with the team ID 
```
      "tagNames": ["SI", "team-15"],
```

@jonathancesario

Comment (tonoman3g) [reply sample/SI/config_si.json]: @michael-jong-ds please help add above tags

Comment (michael-jong-ds) [reply sample/SI/config_si.json]: Added

**Pull request merged**